src/component/archive/archive-list-component.jsx

```diff
@@ -14,38 +16,117 @@ class ArchiveList extends Component {
     componentDidMount() {
         this.props.fetchArchive();
     }
+    renderStrategyDetail(feature) {
```

Comment:
**Minor:** You can choose to introduce sub-components, dedicated to render this part, instead of having these `render*()` functions. This will make the `render`-function a bit more readable.